Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -491,6 +491,10 @@ public static InputActionAsset FromJson(string json)
         /// An exception is if, of the multiple actions with the same name, some are enabled and some are disabled. In
         /// this case, the first action that is enabled is returned.
         ///
+        /// If an action name contains a slash "/", e.g. "yaw/pitch" and there is also a map called "yaw" which
+        /// contains an action "pitch", the action "pitch" within the map "jump" will be returned instead of the
```

Comment:
I think I have some problems with text search today because it tells me `jump` is mentioned here for the first time.

Reply:
Addressed in https://github.com/Unity-Technologies/InputSystem/pull/2160/commits/b78188bc492ef186ddfc4a9ec4d6e81cb547d813